server-spi/src/main/java/org/keycloak/sessions/AuthenticationSessionProvider.java

```diff
@@ -71,6 +72,11 @@ default RootAuthenticationSessionModel createRootAuthenticationSession(String id
      */
     void removeRootAuthenticationSession(RealmModel realm, RootAuthenticationSessionModel authenticationSession);
 
+    /**
+     * Remove expired events in all the realms
```

Comment:
Should be here rather something like "Remove expired authentication sessions in all the realms" ?

Reply:
Done